src/test/java/uk/gov/hmcts/reform/userprofileapi/domain/IdamRegistrationInfoTest.java

```diff
@@ -48,32 +41,22 @@ public void testOneArgConstructor() {
 
     @Test
     public void isSuccessFromIdam() {
-        when(httpStatusMock.is2xxSuccessful()).thenReturn(true);
 

```

Comment:
can you prefix test to all missing test cases

Reply:
done